### PR TITLE
EWPP-33: Add Documents shared field on Project content type

### PR DESCRIPTION
**Pull request by drishu**
### Description

Add oe_documents to project.

### Change log

- Added: oe_documents to top group & default display

### CODE REVIEW

**Review by nagyad — CHANGES_REQUESTED** [review]
Missing tests! Please, extend the project rendering test with the document field.

File: tests/Functional/ContentProjectRenderTest.php [hunk 1]
```diff
@@ -56,21 +56,36 @@ protected function setUp() {
    */
   public function testProjectRendering(): void {
     // Create a document for Project results.
-    $file = file_save_data(file_get_contents(drupal_get_path('module', 'oe_media') . '/tests/fixtures/sample.pdf'), 'public://test.pdf');
-    $file->setPermanent();
-    $file->save();
+    $file1 = file_save_data(file_get_contents(drupal_get_path('module', 'oe_media') . '/tests/fixtures/sample.pdf'), 'public://test.pdf');
```
Comment (nagyad) [inline tests/Functional/ContentProjectRenderTest.php]: $file_1

File: tests/Functional/ContentProjectRenderTest.php [hunk 2]
```diff
@@ -56,21 +56,36 @@ protected function setUp() {
...
+    $file1->setPermanent();
+    $file1->save();
 
-    $media = $this->getStorage('media')->create([
+    $media1 = $this->getStorage('media')->create([
```
Comment (nagyad) [inline tests/Functional/ContentProjectRenderTest.php]: $media_1

File: tests/Functional/ContentProjectRenderTest.php [hunk 3]
```diff
@@ -56,21 +56,36 @@ protected function setUp() {
...
+        'target_id' => (int) $file1->id(),
       ],
       'uid' => 0,
       'status' => 1,
     ]);
+    $media1->save();
 
-    $media->save();
+    // Create a document for Documents.
+    $file2 = file_save_data(file_get_contents(drupal_get_path('module', 'oe_media') . '/tests/fixtures/sample.pdf'), 'public://document.pdf');
```
Comment (nagyad) [inline tests/Functional/ContentProjectRenderTest.php]: $file_2

File: tests/Functional/ContentProjectRenderTest.php [hunk 4]
```diff
@@ -56,21 +56,36 @@ protected function setUp() {
...
+    $file2->setPermanent();
+    $file2->save();
+
+    $media2 = $this->getStorage('media')->create([
```
Comment (nagyad) [inline tests/Functional/ContentProjectRenderTest.php]: $media_2

File: tests/Functional/ContentProjectRenderTest.php [hunk 5]
```diff
@@ -181,6 +201,19 @@ public function testProjectRendering(): void {
...
+    $file_row = $file_wrapper->find('css', '.ecl-file .ecl-file__container');
+    $file_title = $file_row->find('css', '.ecl-file__title');
+    $this->assertContains('Test document 2', $file_title->getText());
+    $file_info_language = $file_row->find('css', '.ecl-file__info div.ecl-file__language');
+    $this->assertContains('English', $file_info_language->getText());
+    $file_info_properties = $file_row->find('css', '.ecl-file__info div.ecl-file__meta');
+    $this->assertContains('KB - PDF)', $file_info_properties->getText());
+    $file_download_link = $file_row->find('css', '.ecl-file__download');
+    $this->assertContains('/document.pdf', $file_download_link->getAttribute('href'));
+    $this->assertContains('Download', $file_download_link->getText());
```
Comment (nagyad) [inline tests/Functional/ContentProjectRenderTest.php:222]: I miss the file size from the assertions.

**Review by nagyad — DISMISSED** [review]

File: tests/Functional/ContentProjectRenderTest.php (hunk 1 above)
Comment (drishu) [reply tests/Functional/ContentProjectRenderTest.php]: done

File: tests/Functional/ContentProjectRenderTest.php (hunk 2 above)
Comment (drishu) [reply tests/Functional/ContentProjectRenderTest.php]: done

File: tests/Functional/ContentProjectRenderTest.php (hunk 3 above)
Comment (drishu) [reply tests/Functional/ContentProjectRenderTest.php]: done

File: tests/Functional/ContentProjectRenderTest.php (hunk 4 above)
Comment (drishu) [reply tests/Functional/ContentProjectRenderTest.php]: done

File: tests/Functional/ContentProjectRenderTest.php (hunk 5 above)
Comment (drishu) [reply tests/Functional/ContentProjectRenderTest.php:222]: done

**Review by imanoleguskiza — DISMISSED** [review]

**Review by imanoleguskiza — DISMISSED** [review]

**Review by ademarco — APPROVED** [review]

**Review by imanoleguskiza — APPROVED** [review]

**Pull request merged**